package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-dashboard",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
What is the policy on the version ,  we usually change the last digit for small changes

Reply:
True, I considered adding authentication and logging in to be a larger change than usual. We don't have an agreed upon set of rules for increasing version numbers. Happy to change it if you want?